hoomd/ForceCompute.h

```diff
@@ -64,6 +65,94 @@ class PYBIND11_EXPORT ForceCompute : public Compute
         m_deltaT = dt;
         }
 
+    template<class Real> struct vec6
```

Comment:
```suggestion
    namespace detail {
    template<class Real> struct vec6
```
And a closing brace at the end of the class. This makes it clear that the class is a detail of the implementation and not part of the public API.